main-app/app/scripts/bingoTicket/bingoModel.js

```diff
@@ -6,32 +6,47 @@
             me.tickets = [];
             me.bingoStrip = [];
 
-            var clearOldTickets = function () {
-                me.tickets = [];
-                me.bingoStrip = [];
-            };
-
             me.getTicket = function (ticketFromServer) {
                 clearOldTickets();
                 generateTickets(ticketFromServer);
-                for (var i = 0; i < me.tickets.length; i++) {
+                for(var i = 0; i < me.tickets.length; i++){
                     createBingoTicketGrid();
                     splitingNumbersUp(me.tickets[i]);
                 }
             };
 
+            var clearOldTickets = function () {
+                me.tickets = [];
+                me.bingoStrip = [];
+            };
+
+            var BingoCell = function () {
+                this.bingoNumber = null;
+                this.cssClass = "unmarked";
```

Comment:
This should be a boolean possibly called `numberCalled`.
You will probably find a function `isToGo()` - this returns false if the number is null, else `!this.bingoNumber`